cmd/check.go

```diff
@@ -27,6 +27,9 @@ func init() {
 		"due to the rate limit imposed for unauthenticated requests. (env: PFLT_DOCKERCONFIG)")
 	viper.BindPFlag("dockerConfig", checkCmd.PersistentFlags().Lookup("docker-config"))
 
+	checkCmd.Flags().String("artifacts", "", "Where check-specific artifacts will be written. (env: PFLT_ARTIFACTS)")
+	viper.BindPFlag("artifacts", checkCmd.Flags().Lookup("artifacts"))
```

Comment:
This needs to be PersistentFlags(), like docker-config above it. Otherwise, it doesn't show in help for check container or check operator, only check --help.